Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <PropertyGroup Label="Package Versions">
+    <NpgsqlVersion>4.0.3</NpgsqlVersion>
```

Comment:
We can maybe wait with this PR for two more days, and then merge it with 4.0.4 instead?

Reply:
Sure, no rush on my end.

Reply:
Sure, no rush on my end.

Reply:
Unresolving to remember to change this after the 4.0.4 release.

Reply:
Oh you already made the change, ignore...